simulator-samples/sample-swagger/src/test/java/org/citrusframework/simulator/ResponseGeneratorIT.java

```diff
@@ -0,0 +1,187 @@
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.not;
+
+import org.citrusframework.annotations.CitrusTest;
+import org.citrusframework.container.BeforeSuite;
+import org.citrusframework.container.SequenceBeforeSuite;
+import org.citrusframework.dsl.endpoint.CitrusEndpoints;
+import org.citrusframework.http.client.HttpClient;
+import org.citrusframework.simulator.sample.Simulator;
+import org.citrusframework.spi.Resources;
+import org.citrusframework.testng.spring.TestNGCitrusSpringSupport;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.ContextConfiguration;
+import org.testng.annotations.Test;
+
+@Test
+@ContextConfiguration(classes = ResponseGeneratorIT.EndpointConfig.class)
+public class ResponseGeneratorIT extends TestNGCitrusSpringSupport {
+
+    @Autowired
+    @Qualifier("pingClient")
+    private HttpClient pingClient;
+
+    @CitrusTest
+    public void shouldPerformDefaultOpenApiPingOperation() {
+        variable("id", "1234");
+
+        $(http().client(pingClient)
+            .send()
+            .put("/ping/${id}")
+            .message()
+            .accept(MediaType.APPLICATION_JSON_VALUE)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .body(new Resources.ClasspathResource("templates/ping.json")));
+
+        $(http().client(pingClient)
+            .receive()
+            .response(HttpStatus.OK)
+            .message()
+            .header("Ping-Time", "@isNumber()@")
+            .validate(jsonPath()
+                .expression("$.pingCount", not(equalTo("1001")))));
+    }
+
+    @CitrusTest
+    public void shouldPerformSpecificApiPingOperation() {
+        long currentTime = System.currentTimeMillis();
+        long expectedPingLimit = currentTime -  1L;
+
+        variable("id", "5000");
+
+        $(http().client(pingClient)
+            .send()
+            .put("/ping/${id}")
+            .message()
+            .accept(MediaType.APPLICATION_JSON_VALUE)
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .body(new Resources.ClasspathResource("templates/ping.json")));
+
+        $(http().client(pingClient)
+            .receive()
+            .response(HttpStatus.OK)
+            .message()
+            .header("Ping-Time", "@isNumber()@")
+            .validate(jsonPath()
+                .expression("$.id", "5000")
+                .expression("$.pingCount", "@greaterThan("+expectedPingLimit+")@"))
```

Comment:
I think this file needs a `ctrl`+`alt` + `l`. there are some static imports that could be adjusted as well, e.g. `MediaType` and `HttpStatus`.